src/pages/Governance/VotingWallet/index.tsx

```diff
@@ -84,47 +89,69 @@ export const VotingWalletUi: React.FC<VotingWalletUiProps> = ({
 
   const previouslyDelegated = !!delegate;
   const userHasLockedXVS = userStakedMantissa.isGreaterThan(0);
+  const showDepositXvs = !isDataLoading && connectedWallet && !userHasLockedXVS;
+  const showDelegateButton =
+    !isDataLoading && connectedWallet && userHasLockedXVS && voteProposalFeatureEnabled;
+  const shouldApplyMarginToTotalLocked =
+    voteProposalFeatureEnabled || !connectedWallet || showDepositXvs;
+
   return (
-    <div css={styles.root}>
-      <Typography variant="h4">{t('vote.votingWallet')}</Typography>
+    <div className="flex flex-1 flex-col lg:ml-4">
+      <h4 className="m-0 text-lg font-semibold leading-normal">{t('vote.votingWallet')}</h4>
```

Comment:
`leading-normal` corresponds to the line-height that's already applied to all text elements, so you don't need to add it here